backend/apis/clients/list-clients.api.js

```diff
@@ -101,7 +101,7 @@ function clientListQuery(query, pageNum, pageSize) {
   }
 
   let joinIntakeServices = "";
-  if (query.program || query.service) {
+  if (query.programInterest || query.serviceInterest) {
```

Comment:
I renamed `program` to `programInterest` now that we have two different kinds of program-related searches - (1) interest, and (2) interaction received